py/h2o_wave/server.py

```diff
@@ -73,6 +73,21 @@ def __init__(self, username: str, subject: str, access_token: str, refresh_token
         """The access token of the user."""
         self.refresh_token = refresh_token
         """The refresh token of the user."""
+        self._session_id = session_id
+        """Session identifier. Do not access, internal use only."""
+    
+    async def force_token_refresh(self):
+        """
+        Implictly refresh OIDC tokens when needed, e.g. during long-running background jobs.
```

Comment:
Explicitly IMHO.

Implicit is something that is implied, explicit is something that us stated directly.

We need to call the endpoint directly, therefore I think the refresh is done "explicitly".

Reply:
Agree.